src/Driver/WpcliDriver.php

```diff
@@ -38,11 +43,18 @@ class WpcliDriver extends BaseDriver
      * @param string $path  Absolute path to WordPress site's files. This or $alias must be not falsey.
      * @param string $url   WordPress site URL.
      */
-    public function __construct($alias, $path, $url)
+    public function __construct($alias, $path, $url, $binary = null)
```

Comment:
* Needs PHPDoc for the new parameter.
* Not sure it needs to have an optional value, we could just document in the PHPDoc that it has two types (string and null). This assumes Symfony DI is passing "null" to the constructor as opposed to not passing it at all (in which case this makes sense!!).